### PR TITLE
milestone1 build

### PR DESCRIPTION
**Pull request by DVK02**

### CODE REVIEW

File: Assets/Scripts/PromptController.cs
Comment (pecasdev) [inline Assets/Scripts/PromptController.cs:1]: what is the purpose of this file?

**Review by Ovven999 — APPROVED** [review]
looks good and the iss able to be played

File: Assets/Scripts/PromptController.cs
Comment (DVK02) [reply Assets/Scripts/PromptController.cs:1]: Oops, I totally forgot I created that. I'll get a new PR going

Comment (pecasdev) [reply Assets/Scripts/PromptController.cs:1]: new PR? just change the file and recommit it to this PR

Comment (DVK02) [reply Assets/Scripts/PromptController.cs:1]: errr, that's what i meant :P

**Review by pecasdev — APPROVED** [review]

**Pull request merged**